DSharpPlus/Entities/DiscordMessage.cs

```diff
@@ -309,13 +315,13 @@ internal DiscordMessageReference InternalBuildMessageReference()
                     Discord = client
                 };
 
-            var channel = client.InternalGetCachedChannel(channelId);
+            var channel = client.InternalGetCachedChannel((ulong)channelId);
```

Comment:
Since a channel id can now be null, attempting to cast this will throw an exception if it's null.

Reply:
Right. My apologies.